src/common/post.ts

```diff
@@ -1,14 +1,17 @@
-import { DataSource } from 'typeorm';
+import { DataSource, EntityManager } from 'typeorm';
 import fetch from 'node-fetch';
-import { User } from '../entity';
+import { FreeformPost, SquadSource, User, WelcomePost } from '../entity';
 import { Comment, ExternalLinkPreview, Post } from '../entity';
 import { ValidationError } from 'apollo-server-errors';
 import { isValidHttpUrl } from './links';
+import { markdown } from './markdown';
+import { generateShortId } from '../ids';
 
 export const defaultImage = {
   urls: process.env.DEFAULT_IMAGE_URL?.split?.(',') ?? [],
   ratio: parseFloat(process.env.DEFAULT_IMAGE_RATIO),
   placeholder: process.env.DEFAULT_IMAGE_PLACEHOLDER,
+  welcomePost: process.env.WELCOME_POST_BANNER,
```

Comment:
Please make sure that we use proper cloudinary transformations.
Can you drop here the link you use?

Reply:
It should be the link below:
https://daily-now-res.cloudinary.com/image/upload/f_auto/public/welcome_post

Reply:
Let's use this: https://daily-now-res.cloudinary.com/image/upload/f_auto,q_auto/public/welcome_post